apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/CarrierClaimTransformerTest.java

```diff
@@ -27,10 +26,26 @@
 import org.hl7.fhir.dstu3.model.ExplanationOfBenefit.ItemComponent;
 import org.hl7.fhir.dstu3.model.codesystems.ClaimCareteamrole;
 import org.hl7.fhir.exceptions.FHIRException;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 /** Unit tests for {@link gov.cms.bfd.server.war.stu3.providers.CarrierClaimTransformer}. */
 public final class CarrierClaimTransformerTest {
+  /** The Metric Registry to use for the test. */
+  private MetricRegistry metricRegistry;
+  /** The FDA drug lookup to use for the test. */
+  private FdaDrugCodeDisplayLookup drugDisplayLookup;
+  /** The NPI org lookup to use for the test. */
+  private NPIOrgLookup npiOrgLookup;
+
+  /** One-time setup of objects that are normally injected. */
+  @BeforeEach
+  protected void setup() {
+    metricRegistry = new MetricRegistry();
+    drugDisplayLookup = FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting();
+    npiOrgLookup = new NPIOrgLookup();
```

Comment:
These should be mocked, not instantiated. Here and everywhere else this was done in the unit tests. I apparently missed fixing the mocks in the claim type unit tests in v2, so you may have been misled; please look at something like R4PatientResourceProviderTest for an example (the transformer tests should be much easier to mock however, since they only have limited, simple dependencies; you may just get away with setting up the mocks and leaving it as that)

Reply:
Well that just seems messed up. Can we just fix that mess so we can mock things correctly? (Also can you show me where it is? Maybe I can do a little fiddling with it for better direction)

Reply:
Metric registry can be mocked, but the other two deferred to https://jira.cms.gov/browse/BFD-2766